ospfd/ospf_apiserver.c

```diff
@@ -1343,6 +1353,130 @@ int ospf_apiserver_handle_sync_lsdb(struct ospf_apiserver *apiserv,
 	return rc;
 }
 
+/*
+ * -----------------------------------------------------------
+ * Followings are functions for synchronization.
+ * -----------------------------------------------------------
+ */
+
+int ospf_apiserver_handle_sync_reachable(struct ospf_apiserver *apiserv,
+					 struct msg *msg)
+{
+	struct ospf *ospf = ospf_lookup_by_vrf_id(VRF_DEFAULT);
+	struct route_table *rt = ospf->all_rtrs;
+	uint32_t seqnum = msg_get_seq(msg);
+	struct in_addr *a, *abuf;
+	struct msg_reachable_change *areach;
+	struct msg *amsg;
+	uint mcount, count;
+	int _rc, rc = 0;
+
+	if (!rt)
+		goto out;
```

Comment:
why don't we just put the return here rather than using the goto?

Reply:
A reply message is sent after `out:`, and some state is modified.